examples/2_eth_stats_sample/dozer-config.yaml

```diff
@@ -23,13 +23,26 @@ connections:
     name: eth_conn
 
 sql: |
-  select punkIndex, minValue, toAddress
-  INTO punk_offered from punks_PunkOffered;
+  SELECT
```

Comment:
I think we should add comments for each of these too. So users can understand when they see it.

Reply:
ok